src/smc-webapp/project_new.cjsx

```diff
@@ -272,6 +276,25 @@ ProjectNew = (name) -> rclass
             on_error     : on_error
             switch_over  : true
 
+    decline_file : ->
+        @setState(warning : false)
+        @refs.project_new_filename.getInputDOMNode().disabled = false
```

Comment:
This leads me to believe my hunch from before is true. Just change the ref component to render based on `@state.warning` instead of disabling it via the DOM.